heartbeat/monitors/mocks_test.go

```diff
@@ -99,10 +99,10 @@ func createMockJob(name string, cfg *common.Config) ([]Job, error) {
 }
 
 func mockPluginBuilder() pluginBuilder {
-	return pluginBuilder{"test", ActiveMonitor, func(s string, config *common.Config) ([]Job, error) {
+	return pluginBuilder{"test", ActiveMonitor, func(s string, config *common.Config) ([]Job, int, error) {
 		c := common.Config{}
 		j, err := createMockJob("test", &c)
```

Comment:
I think this breaks the tests because there is not monitor type `test`.

Reply:
This is now fixed since we only log instead of panic in this case.

Reply:
This is now further improved by our dynamic creation of metrics, so this scenario is no longer possible.